frontend/src/components/admin/assessment-creation/QuestionCard.tsx

```diff
@@ -134,35 +133,23 @@ const QuestionCard = ({
               Question {index + 1}
             </Text>
             <Spacer />
-            <Box
-              _hover={{ color: "grey.300" }}
-              color="blue.300"
-              cursor="pointer"
-              fontSize="24px"
-              paddingRight="1"
-            >
-              <Button
-                as={IconButton}
-                color="currentColor"
-                fontSize="24px"
+            <HStack spacing={1}>
```

Comment:
```suggestion
            <HStack spacing={4}>
```
looks a little wider originally